src/datastore/fetchingQueries.ts

```diff
@@ -34,6 +34,14 @@ export const fetchUserRiddleInfos = async (
 	return infoRes.docs.length > 0 ? infoRes.docs : undefined; //User may not have answered yet, so there is no record
 };
 
+export const fetchRiddleSolvers = async (linkId: string) => {
```

Comment:
Mohli bychom sjednotit s `fetchUserRiddleInfos` s tím, že by tato funkce měla `userEmail` jako volitelný parametr.

Reply:
Asi bych se klonil k tomu to nechat takto, protože `fetchUserRiddleInfos` přijímá `linkIds` (pole) a `fetchRiddleSolvers` přijímá `linkId` (string). I návratové hodnoty jsou jiné. Bylo by tam dost ifů, tak bych to možná raději nechal takto.

Reply:
ok toho rozdílu ve vstupech jsem si nevšiml, tím pádem jich asi začíná být dost těch rozdílů.
s tou návratovou hodnotou je to ale dobrý point, neměl `fetchUserRiddleInfos ` by dost možná nemělo vracet undefined, aby to bylo konzistentní, ještě se na to po mergi podívám.